cpp/test/IceSSL/configuration/SchannelTests.cpp

```diff
@@ -116,7 +116,8 @@ createServer(ServerAuthenticationOptions serverAuthenticationOptions, TestHelper
 Ice::CommunicatorPtr
 createClient(optional<ClientAuthenticationOptions> clientAuthenticationOptions = nullopt)
 {
-    return initialize(Ice::InitializationData{.clientAuthenticationOptions = clientAuthenticationOptions});
+    return initialize(Ice::InitializationData{
+        .clientAuthenticationOptions = clientAuthenticationOptions.value_or(ClientAuthenticationOptions{})});
```

Comment:
Using `ClientAuthenticationOptions{}` and `nullopt` are a bit different:

- `ClientAuthenticationOptions{}` use the default options.
- `nullopt` uses the default IceSSL properties configuration.